### PR TITLE
feat: RatingIcon story

### PR DESCRIPTION
**Pull request by notrab**
Not sure why but icons show up both states, need to ask @FrancoAguzzi what's up.

### CODE REVIEW

File: apps/storybook/stories/Nameguard/RatingIcon.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import * as RatingIconStories from './RatingIcon.stories';
+import { RatingIcon } from '@namehash/nameguard-react';
+
+<Meta of={RatingIconStories} />
+
+# RatingIcon
+
+<IconGallery>
+  <IconItem name="pass">
+    <RatingIcon rating="pass" />
```
Comment (FrancoAguzzi) [inline apps/storybook/stories/Nameguard/RatingIcon.mdx:11]: @notrab best to import `Rating` from `@namehash/nameguard`

File: apps/storybook/stories/Nameguard/RatingIcon.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  </IconItem>
+  <IconItem name="warn">
+    <RatingIcon rating="warn" />
```
Comment (FrancoAguzzi) [inline apps/storybook/stories/Nameguard/RatingIcon.mdx:14]: Same here

File: apps/storybook/stories/Nameguard/RatingIcon.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  </IconItem>
+  <IconItem name="alert">
+    <RatingIcon rating="alert" />
```
Comment (FrancoAguzzi) [inline apps/storybook/stories/Nameguard/RatingIcon.mdx:17]: And here

**Comment by FrancoAguzzi** [comment]
> Not sure why but icons show up both states, need to ask @FrancoAguzzi what's up.

Which states?

**Comment by FrancoAguzzi** [comment]
@notrab nice tech included in the latest commits

**Review by FrancoAguzzi — APPROVED** [review]

**Review by lightwalker-eth — COMMENTED** [review]
@notrab Really appreciate this update. Shared feedback in Shortcut: https://app.shortcut.com/ps-web3/story/25221/refine-storybook-for-ratingicon#activity-25229

**Pull request merged**